INFO     core.jenkins_utils:jenkins_utils.py:197 running cleanup
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 403 None
DEBUG    core.jenkins_utils:jenkins_utils.py:86 failed to get page on the first try, retrying after login
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /login?from=%2F HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /j_spring_security_check HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /job/Freestyle_Project/doDelete HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/securityRealm/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /computer/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/credentials/store/system/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /me/my-views/view/all/submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /user/admin/appearance/configSubmit HTTP/1.1" 200 132
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /user/admin/security/ HTTP/1.1" 200 None
DEBUG    conftest:conftest.py:57 Argument --remote-allow-origins=* added to the chrome
DEBUG    conftest:conftest.py:57 Argument --disable-gpu added to the chrome
DEBUG    conftest:conftest.py:57 Argument --no-sandbox added to the chrome
DEBUG    conftest:conftest.py:57 Argument --disable-dev-shm-usage added to the chrome
DEBUG    conftest:conftest.py:57 Argument --headless=new added to the chrome
DEBUG    conftest:conftest.py:57 Argument --window-size=1920,1080 added to the chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:101 Selenium Manager binary found at: /opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:114 Executing process: /opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager --browser chrome --debug --language-binding python --output json
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Found chromedriver 136.0.7103.92 in PATH: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 chrome detected at /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Running command: /usr/bin/google-chrome --version
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Output: "Google Chrome 136.0.7103.92 "
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Detected browser: chrome 136.0.7103.92
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Required driver: chromedriver 136.0.7103.94
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Driver path: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Browser path: /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.service:service.py:231 Started executable: `/usr/bin/chromedriver` in a child process with pid: 13079 using 0 to output -3
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:48993
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48993 "POST /session HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48993 "POST /session/a8d5d9715fcd0f8f7bc6870349ac3ab7/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48993 "GET /session/a8d5d9715fcd0f8f7bc6870349ac3ab7/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48993 "POST /session/a8d5d9715fcd0f8f7bc6870349ac3ab7/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48993 "POST /session/a8d5d9715fcd0f8f7bc6870349ac3ab7/element/f.6BD2A7FFFB8A32AEFFEBF0D6A34EDED7.d.BCD55C5AD09788A5279CADB4C236B87B.e.1/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48993 "POST /session/a8d5d9715fcd0f8f7bc6870349ac3ab7/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48993 "POST /session/a8d5d9715fcd0f8f7bc6870349ac3ab7/element/f.6BD2A7FFFB8A32AEFFEBF0D6A34EDED7.d.BCD55C5AD09788A5279CADB4C236B87B.e.3/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48993 "POST /session/a8d5d9715fcd0f8f7bc6870349ac3ab7/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48993 "POST /session/a8d5d9715fcd0f8f7bc6870349ac3ab7/element/f.6BD2A7FFFB8A32AEFFEBF0D6A34EDED7.d.BCD55C5AD09788A5279CADB4C236B87B.e.9/click HTTP/1.1" 200 0
DEBUG    MainPage:ui_element.py:26 waiting for visibility_of_element_located(('link text', 'Manage Jenkins'),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48993 "POST /session/a8d5d9715fcd0f8f7bc6870349ac3ab7/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48993 "POST /session/a8d5d9715fcd0f8f7bc6870349ac3ab7/execute/sync HTTP/1.1" 200 0
DEBUG    MainPage:ui_element.py:26 waiting for visibility_of_element_located(('link text', 'Manage Jenkins'),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48993 "POST /session/a8d5d9715fcd0f8f7bc6870349ac3ab7/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48993 "POST /session/a8d5d9715fcd0f8f7bc6870349ac3ab7/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48993 "GET /session/a8d5d9715fcd0f8f7bc6870349ac3ab7/cookie HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /crumbIssuer/api/json HTTP/1.1" 200 155
DEBUG    core.jenkins_utils:jenkins_utils.py:26 update_crumb: c3f7f69b9f0124cc55cf2b3393c6e3330175705b3ae58f0af46e32436a590151
DEBUG    LoginPage:login_page.py:32 login crumb: c3f7f69b9f0124cc55cf2b3393c6e3330175705b3ae58f0af46e32436a590151
DEBUG    MainPage:ui_element.py:26 waiting for element_to_be_clickable(('link text', 'New Item'),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48993 "POST /session/a8d5d9715fcd0f8f7bc6870349ac3ab7/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48993 "POST /session/a8d5d9715fcd0f8f7bc6870349ac3ab7/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48993 "GET /session/a8d5d9715fcd0f8f7bc6870349ac3ab7/element/f.6BD2A7FFFB8A32AEFFEBF0D6A34EDED7.d.E7EACFFA86ABCA73E7A846D7759BACEA.e.48/enabled HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48993 "POST /session/a8d5d9715fcd0f8f7bc6870349ac3ab7/element/f.6BD2A7FFFB8A32AEFFEBF0D6A34EDED7.d.E7EACFFA86ABCA73E7A846D7759BACEA.e.48/click HTTP/1.1" 200 0
DEBUG    NewItemPage:ui_element.py:26 waiting for visibility_of_element_located(('xpath', "//h1[text()='New Item']"),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48993 "POST /session/a8d5d9715fcd0f8f7bc6870349ac3ab7/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48993 "POST /session/a8d5d9715fcd0f8f7bc6870349ac3ab7/execute/sync HTTP/1.1" 200 0
DEBUG    NewItemPage:ui_element.py:26 waiting for visibility_of_element_located(('xpath', "//h1[text()='New Item']"),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48993 "POST /session/a8d5d9715fcd0f8f7bc6870349ac3ab7/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48993 "POST /session/a8d5d9715fcd0f8f7bc6870349ac3ab7/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48993 "GET /session/a8d5d9715fcd0f8f7bc6870349ac3ab7/title HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:48993 "DELETE /session/a8d5d9715fcd0f8f7bc6870349ac3ab7 HTTP/1.1" 200 0